src/Apps/ArtQuiz/Components/ArtQuizMain.tsx

```diff
@@ -80,27 +90,47 @@ export const ArtQuizMain: FC<ArtQuizMainProps> = () => {
           </RouterLink>
         </Flex>
 
-        <Flex justifyContent="center" alignItems="center" flex={1}>
-          <Box
-            width={400}
-            height={400}
-            bg="black10"
-            display="flex"
-            alignItems="center"
-            justifyContent="center"
-          >
-            <Text variant="xxl">{index + 1}</Text>
-          </Box>
+        <Flex
+          position="relative"
+          justifyContent="center"
+          alignItems="center"
+          flex={1}
+          flexDirection="column"
+        >
+          {cards.map((card, i) => {
+            const mode = i === activeIndex ? Mode.Active : card.mode
+
+            return (
+              <ArtQuizCard key={card.id} mode={mode} position="absolute">
```

Comment:
I'll want to replace these with sample images (including retina sized srcSets) to see if the performance changes.